package.json

```diff
@@ -39,19 +39,19 @@
     "typescript": "^2.6.2"
   },
   "dependencies": {
-    "@types/bluebird": "3.5.18",
+    "@types/bluebird": "3.5.20",
     "@types/dockerode": "2.5.5",
-    "@types/lodash": "^4.14.87",
-    "@types/tar-stream": "^1.6.0",
+    "@types/lodash": "4.14.112",
+    "@types/tar-stream": "1.6.0",
```

Comment:
Pinning `@types` packages to specific versions (removing the caret prefix): I don't know if this a settled best practice, but I've seen past pull requests aiming at doing specifically that, avoid typescript compilation breaking unexpectedly.